src/components/screens/monitor/transactions/transactions.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import TransactionsTable from '../../../shared/transactionsTable';
+import PageLayout from '../../../toolbox/pageLayout';
+
+const Transactions = ({ t, transactions }) => {
+  const columns = [
+    { header: 'Sender', className: `${grid['col-xs-3']}`, key: 'senderId' },
+    { header: 'Recipient', className: `${grid['col-xs-3']}`, key: 'recipientId' },
+    {
+      header: 'Date', className: `${grid['col-xs-2']}`, key: 'timestamp', isSortingColumn: true, defaultSort: true,
+    },
+    {
+      header: 'Amount', className: `${grid['col-xs-2']}`, key: 'amount', isSortingColumn: true,
+    },
+    { header: 'Fee', className: `${grid['col-xs-1']}  hidden-m`, key: 'fee' },
+    { header: 'Status', className: `${grid['col-xs-1']}`, key: 'confirmations' },
```

Comment:
The UI strings should have `t(...)` on them, because else the i18n scanner won't find them and they won't be added to `common.json`
```suggestion
    { header: t('Status'), className: `${grid['col-xs-1']}`, key: 'confirmations' },
```

In this case, you don't see a difference, because all these headers are also used somewhere else and scanner gets them from there.

Reply:
Added the `t(...)` to header.